src/DisplayMoviesData.jsx

```diff
@@ -0,0 +1,59 @@
+/* eslint-disable no-unused-vars */
+import React, { Component } from "react";
+import { getMovieIndex } from "./modules/moviesData"
+import { Card, Image, Grid, Container} from 'semantic-ui-react'
+
+class DisplayMoviesData extends Component {
+  state = {
+    moviesData: [],
+  };
+
+  componentDidMount() {
+    this.getMoviesData();
+  }
+
+  async getMoviesData() {
+    let result = await getMovieIndex();
+    this.setState({ moviesData: result });
+  }
+
+  render() {
+    let movieIndexDisplay;
+    if (Array.isArray(this.state.moviesData) && this.state.moviesData.length) {
+      movieIndexDisplay = (
+        <Grid columns={2} data-cy="index">
+          {this.state.moviesData.map(movie => {
+            return (
+              <Grid.Column> 
+                <Card key={movie.id} data-cy={`movie-${movie.id}`}>
```

Comment:
Just extract the `Card` to a separate component and send in the `movie` as props 👍🏻